Podcast/RecommendedEpisodesOuterTableViewCell.swift

```diff
@@ -56,10 +56,10 @@ class RecommendedEpisodesOuterTableViewCell: UITableViewCell, UITableViewDelegat
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath) as? EpisodeTableViewCell else { return UITableViewCell() }
         let episode = dataSource?.recommendedEpisodesTableViewCell(cell: self, dataForItemAt: indexPath) ?? Episode()
-        cell.setupWithEpisode(episode: episode)
+        cell.episodeSubjectView.setupWithEpisode(episode: episode)
```

Comment:
This may be annoying, but it might be nice to leave it as cell.setupWithEpisode to better abstract away what's in the cell.

Reply:
Agreed, was just lazy before but ill change it